ansible/home/ec2-user/.env.j2

```diff
@@ -0,0 +1,16 @@
+  GNU nano 5.8                                                                                             .env.j2                                                                                                        
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_API_TOKEN={{trello_api_token}}
+TRELLO_BOARD_ID=658400a65cbc3ab26e171cb0
+TRELLO_BOARD_LIST_ID_TODO=658400a6795546f35c80c673
+TRELLO_BOARD_LIST_ID_DONE=658400a77be14947511810e0
```

Comment:
This is how I'd solve this exercise, but for a real project we'd probably want to extract these into variables like `{{trello_api_key}}` - we could set these values as defaults in the playbook since they're not sensitive so that we still don't need to type them every time normally, but worth noting that as it stands we can't easily use this template to deploy to multiple environments